test/CMakeLists.txt

```diff
@@ -67,3 +68,6 @@ if(ENABLE_FUZZER)
     add_subdirectory(fuzz)
 endif()
 add_subdirectory(unit)
+if(ENABLE_TUPLE_COMPRESSION)
+    add_subdirectory(enterprise-unit)
+endif()
```

Comment:
I don't like how you add enterprise-unit tests:
- I don't want to copy CMakeLists.txt from the enterprise code to the open-source dir, as you do in the enterprise PR.
- I want to be able to add new enterprise unit tests without patching the open-source repository.

We need to come up with a better solution. Maybe, we could use [`include()`](https://cmake.org/cmake/help/latest/command/include.html) CMake command and pass the path to include via a new CMake variable?

Reply:
Fixed